docs/src/pages/components/tables/CustomPaginationActionsTable.js

```diff
@@ -171,7 +174,6 @@ export default function CustomPaginationActionsTable() {
               }}
               onPageChange={handleChangePage}
               onRowsPerPageChange={handleChangeRowsPerPage}
-              ActionsComponent={TablePaginationActions}
```

Comment:
If we are going to make this breaking change, we should document it (PR description + migration guide).

Reply:
I think let's preserve the current API and don't introduce breaking change yet.